src/Files.App/UserControls/Pane/InfoPane.xaml

```diff
@@ -238,6 +238,94 @@
 					TextAlignment="Center"
 					TextWrapping="Wrap"
 					Visibility="Collapsed" />
+
+				<!--  Drive Details  -->
+				<TextBlock
+					x:Name="DriveFormatAndTypeTextBlock"
+					Margin="12,12,12,24"
+					HorizontalAlignment="Center"
+					IsTextSelectionEnabled="True"
+					Style="{ThemeResource BodyTextBlockStyle}"
+					TextAlignment="Center"
+					TextWrapping="Wrap"
+					Visibility="Collapsed">
+					<Run Text="{x:Bind ViewModel.SelectedDriveItem.Filesystem, Mode=OneWay}" />
+					<Run Foreground="{ThemeResource TextFillColorTertiaryBrush}" Text="{x:Bind ViewModel.SelectedDriveItem.TypeText, Mode=OneWay}" />
+				</TextBlock>
+
+				<TextBlock
+					x:Name="UsedSpaceTextBlock"
+					HorizontalAlignment="Center"
+					Foreground="{ThemeResource SystemAccentColor}"
+					IsTextSelectionEnabled="True"
+					Style="{ThemeResource SubtitleTextBlockStyle}"
+					Text="{x:Bind ViewModel.SelectedDriveItem.UsedSpaceText, Mode=OneWay}"
+					TextAlignment="Center"
+					TextWrapping="Wrap"
+					Visibility="Collapsed" />
+				<controls:StorageBar
+					x:Name="DriveSpaceProgressBar"
+					Margin="8,8,8,8"
+					HorizontalAlignment="Stretch"
+					TrackBarHeight="4"
+					ValueBarHeight="8"
+					Visibility="Collapsed"
+					Value="{x:Bind ViewModel.SelectedDriveItem.PercentageUsed, Mode=OneWay}" />
+				<Grid
+					x:Name="DriveSpaceGrid"
+					Margin="12,0,12,24"
+					Visibility="Collapsed">
+					<Grid.RowDefinitions>
+						<RowDefinition />
+						<RowDefinition />
+					</Grid.RowDefinitions>
+
+					<Grid.ColumnDefinitions>
+						<ColumnDefinition Width="Auto" />
+						<ColumnDefinition />
+						<ColumnDefinition Width="Auto" />
+					</Grid.ColumnDefinitions>
+
+					<TextBlock
+						x:Name="AvailableSpaceTextBlock"
+						FontFamily="{ThemeResource ContentControlThemeFontFamily}"
+						FontSize="14"
+						IsTextSelectionEnabled="True"
+						Style="{ThemeResource BodyStrongTextBlockStyle}"
+						Text="{x:Bind ViewModel.SelectedDriveItem.FreeSpaceText, Mode=OneWay}"
+						TextAlignment="Left"
+						TextWrapping="Wrap" />
+					<TextBlock
+						x:Name="TotalDriveSpaceTextBlock"
+						Grid.Column="2"
+						Foreground="{ThemeResource TextFillColorTertiaryBrush}"
+						IsTextSelectionEnabled="True"
+						Style="{ThemeResource BodyStrongTextBlockStyle}"
+						Text="{x:Bind ViewModel.SelectedDriveItem.MaxSpaceText, Mode=OneWay}"
+						TextAlignment="Right"
+						TextWrapping="Wrap" />
+					<TextBlock
+						x:Name="AvailableSpaceLabel"
+						Grid.Row="1"
+						HorizontalAlignment="Left"
+						IsTextSelectionEnabled="True"
```

Comment:
Is the header supposed to have text selection?

Reply:
I don't think so, I'll remove it